### PR TITLE
allow to set kpconv params from yaml file.

### PR DESCRIPTION
**Pull request by sidml**
This commit should allow user to configure KPConvLayer and KPConvDeformableLayer by modifying the yaml file.
https://github.com/nicolas-chaulet/torch-points3d/issues/368

### CODE REVIEW

File: torch_points3d/applications/conf/kpconv/unet_4.yaml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class: kpconv.KPConvPaper
 conv_type: "PARTIAL_DENSE"
+KPConvLayer:
```
Comment (tchaton) [inline torch_points3d/applications/conf/kpconv/unet_4.yaml:3]: Thanks for contributing. I need to check, but I think there is another solution to make this work. The unet should be model agnostic.

Comment (tchaton) [reply torch_points3d/applications/conf/kpconv/unet_4.yaml:3]: Let me check on my side. @nicolas-chaulet Do you confirm there is a better way ?

Comment (nicolas-chaulet) [reply torch_points3d/applications/conf/kpconv/unet_4.yaml:3]: Yes, there is, let me create a PR for that, it will just be faster

**Comment by nicolas-chaulet** [comment]
See #378

**Pull request closed without merging**